test/behavior/asm.zig

```diff
@@ -191,3 +191,13 @@ test "asm modifiers (AArch64)" {
     );
     try expectEqual(2 * x, double);
 }
+
+test "SPIR-V string constants with c constraint" {
+    if (builtin.zig_backend != .stage2_spirv) return error.SkipZigTest;
+    if (builtin.target.os.tag != .opencl) return error.SkipZigTest;
+
+    _ = std.gpu.printf("testing printf\n", .{});
+    var a: i32 = -10;
+    _ = &a;
+    _ = std.gpu.printf("a: %d, a + 10: %d\n", .{a, a + 10});
+}
```

Comment:
I have decided to go on with std.gpu.printf for now, so printf suddenly does not get broken next time